tox.ini

```diff
@@ -33,8 +35,8 @@ deps=
 sitepackages = False
 recreate = True
 commands =
-    pip install -U celery
-    pip install -U kombu
+    pip install celery
```

Comment:
Why remove `-U` option?
I think it is not safe 🤔 
All package and cached and cannot find backward incompatible changes.
https://github.com/celery/django-celery-results/blob/v1.0.4/.travis.yml#L3

How about `pip install -r requirements/default.txt` ?

Reply:
Ooops :(
`-r{toxinidir}/requirements/default.txt` is defined in `deps`

I think that installations are unnecessary